portal-api/src/v1/users/login.controller.js

```diff
@@ -33,3 +35,12 @@ module.exports = (username, password) =>
       return updateLastLogin(user, sessionIdentifier, () => resolve({ user, tokenObject }));
     });
   });
+
+module.exports.sendSignInLinkEmail = async (emailAddress, firstName, signInLink) => sendEmail(
+  CONSTANTS.EMAIL_TEMPLATE_IDS.SIGN_IN_LINK,
+  emailAddress,
+  {
+    firstName,
+    signInLink,
+  },
+);
```

Comment:
Can we make the "30 minutes" expiry timeframe a parameter as well please @oscar-richardson-softwire? We'll want to make that a constant in our code so we can easily change the expiry time

Reply:
Added